test/cqlpy/test_tablets.py

```diff
@@ -181,7 +181,40 @@ def test_describe_initial_tablets(cql, skip_without_tablets):
             "AND TABLETS = { 'initial' : 1 }"
     with new_test_keyspace(cql, ksdef) as keyspace:
         desc = cql.execute(f"DESCRIBE KEYSPACE {keyspace}")
-        assert "and tablets = {'initial': 1}" in desc.one().create_statement.lower()
+        assert "and tablets = {'enabled': true, 'initial': 1}" in desc.one().create_statement.lower()
+
+
+# Test to ensure metadata is printed when tablets are enabled during schema description, regardless of the use of WITH INTERNALS.
+#
+# Reproduces https://github.com/scylladb/scylladb/issues/22866
+@pytest.mark.parametrize("tablets", [None, True, False])
+@pytest.mark.parametrize("initial", [None, 0, 7])
+def test_describe_tablets(cql, tablets, initial):
+    if not tablets and initial is not None:
+        pytest.skip('The combination of disabled tablets and initial having a value makes no sense')
```

Comment:
That's not the best way to do this - your way will show in the output running 9 tests, with some of them marked as "skipped" which is misleading and unnecessary.
The right way to do this is to have a single parametrized list, not two, listing the pairs you want to check. I would build the list explicitly, e.g., [(True, 0), (True, 7), (False, None)] but you can also use list comprehension to have Python prepare the list of "valid" pairs for you.